scripts/marketing_data_test.py

```diff
@@ -284,12 +295,20 @@ def marketing_dml_test(
     # treat_est_df = treat_df.drop('treat_id', axis=1)
     # Effect of the treatments trained on
     x_cols = list(X.columns.values)
-    t_cols = list(treat_df.columns.values)
-    t_cols.remove('treat_id')
+    t_cols = list(T.columns.values)
     X['x_id']=X.index.values
-    treat_est_combo = X.merge(treat_df, how='cross')
-    logger.info(f"Calculating effects of {len(treat_df)} treatments on audience size {len(X)} "
-                f" total {len(treat_est_combo)} combinations")
+
+    n_sample = 10000
+    if X.shape[0] > n_sample:
+        treat_est_combo = X.sample(n=n_sample).reset_index().merge(treat_df, how='cross')
+    else:
+        treat_est_combo = X.merge(treat_df, how='cross')
```

Comment:
Add if-else condition in case we are down-sampling with less than `n_sample` value. There is probably a cleaner way to do it, but this works for now.